hawc/apps/assessment/exports.py

```diff
@@ -1,9 +1,108 @@
+            "adaf": "adaf",
+            "confidence": "confidence",
+            "duration": "duration",
+            "basis": "basis",
+            "pod_type": "pod_type",
+            "pod_value": "pod_value",
+            "pod_unit": "pod_unit",
+            "uncertainty": "uncertainty_display",
+            "species_studied": "species_studied",
+            "evidence": "evidence",
+            "tumor_type": "tumor_type",
+            "extrapolation_method": "extrapolation_method",
+            "comments": "comments",
+            "extra": "extra",
+            "created": "created",
+            "last_updated": "last_updated",
+        }
 
-class ValuesListExport(FlatFileExporter):
-    def build_df(self) -> pd.DataFrame:
-        return AssessmentValue.objects.get_df()
+    def get_annotation_map(self, query_prefix: str) -> dict:
+        return {
+            "evaluation_type_display": sql_display(
+                query_prefix + "evaluation_type", constants.EvaluationType
+            ),
+            "value_type_display": sql_display(query_prefix + "value_type", constants.ValueType),
+            "uncertainty_display": sql_display(
+                query_prefix + "uncertainty", constants.UncertaintyChoices
+            ),
+        }
+
+    def prepare_df(self, df):
+        return self.format_time(df)
+
+
+class AssessmentExport(ModelExport):
+    def get_value_map(self) -> dict:
+        return {
+            "id": "id",
+            "name": "name",
+            "cas": "cas",
+            "year": "year",
+            "created": "created",
+            "last_updated": "last_updated",
+        }
+
+    def prepare_df(self, df):
+        return self.format_time(df)
+
+
+class AssessmentDetailExport(ModelExport):
+    def get_value_map(self) -> dict:
+        return {
+            "project_type": "project_type",
+            "project_status": "project_status_display",
+            "project_url": "project_url",
+            "peer_review_status": "peer_review_status_display",
+            "qa_id": "qa_id",
+            "qa_url": "qa_url",
+            "report_id": "report_id",
+            "report_url": "report_url",
+            "extra": "extra",
+            "created": "created",
+            "last_updated": "last_updated",
+        }
+
+    def prepare_df(self, df):
+        return self.format_time(df)
+
+    def get_annotation_map(self, query_prefix: str) -> dict:
+        return {
+            "project_status_display": sql_display(
+                query_prefix + "project_status", constants.Status
+            ),
+            "peer_review_status_display": sql_display(
+                query_prefix + "peer_review_status", constants.PeerReviewType
+            ),
+        }
+
+
+class AssessmentExporter(Exporter):
+    def build_modules(self) -> list[ModelExport]:
+        return [
+            AssessmentExport("assessment", "assessment"),
+            AssessmentDetailExport("assessment_detail", "assessment__details"),
```

Comment:
Found a bug if a user tries to filter for an assessment with an AssessmentValue but not AssessmentDetail, you cant filter for that assessment: `AttributeError: Can only use .dt accessor with datetimelike values`.  I think because in this join table the AssessmentDetail `Created` field wont exist... looking into it more.

Reply:
I think we might need a check before casting the datetimes?

```python
df = pd.DataFrame(data=[[1,None,],[None,None]], columns=['a','b'])
>      a     b
> 0  1.0  None
> 1  NaN  None

df['a'].isnull().all()
> False

df['b'].isnull().all()
> True
```